scripts/configs/drone/main.jsonnet

```diff
@@ -174,6 +191,24 @@ local docker_publish(repo, auth, tag, os, arch, version='') =
     ])
     + releaseOnly,
 
+    step(
+      'docker publish (with browser) tags',
+      [
+        '{ echo latest-with-browser,$(eval ./scripts/version)-with-browser ; } > .tags',  // use with-browser tags for docker plugin
```

Comment:
I'd suggest to use a shorter `-browser` suffix instead of `-with-browser`, just for the sake of brevity. I think the short version does not leave much room for ambiguity. WDYT?

Reply:
I agree! I noticed other Docker images like that for `k6` were using `-with-browser`, but I don't mind shortening it for us

Reply:
Well, normally I would want it to be consistent with other releases but because we build a longer tag name for our images, including the commit hash, I think I like keeping it shorter for this case

Reply:
Hmm... that's a good point, consistency is good... I don't see a clear winner but I'm still leaning towards `-browser` for short. Maybe we can ask the rest of the team to see if anyone has a strong opinion?